src/index.js

```diff
@@ -1,6 +1,4 @@
-"use strict"
-
-module.exports = function(Parser) {
+export default function logicalAssignment(Parser) {
   const acorn = Parser.acorn || require("acorn")
```

Comment:
I think we don't want `require` in ESM.

Reply:
Makes sense. Removed that.

Reply:
This breaks compatibility with acorn < 7.

Reply:
So acorn 6 doesn't have the `Parser.acorn` property?

Reply:
It doesn't, that's the whole point of this `require` :)

Reply:
Oh right, I actually forgot about that.

Although I think it's probably fine - feature-wise we're unlikely to make any changes at this point that we'd want to backport to Acorn 6, so users there can keep using existing version of acorn-logical-assignment, whereas new users will use the latest one (after major bump) with support only for Acorn 7+.

Wdyt?

Reply:
Ok, the `require` is back. I think that's fine to have it as a separate issue (if you want to tackle it at all). We primarily block on the ESM migration at this point.